Sources/HAP/Accessories/Lightbulb.swift

```diff
@@ -37,9 +50,32 @@ extension Service {
             maxValue: 360,
             minValue: 0,
             minStep: 1)
+        public let temperature: GenericCharacteristic<Temperature>?
```

Comment:
The HAP docs also lists brightness, saturation and hue as an optional characteristic. The only required characteristic is on. Can you make those other optional characteristics `Optional<>`?

Reply:
Sure can do, I didn't do that because it could break any code which used Lightbulb as it is, but if you're not so worried about that, I think it is the right change to make. I would also suggest defaulting lightbulb to `.white, isDimmable = false`

Reply:
I'm fine with changing behaviour if that means closing the gap with the official spec.